verification/src/compiler/versions_fetcher.rs

```diff
@@ -84,7 +84,7 @@ impl<T: PartialEq + Send + Sync + 'static> VersionsRefresher<T> {
         fetcher: Arc<F>,
         cron_schedule: Schedule,
     ) {
-        log::info!("spawn version refresh job");
+        tracing::info!("spawn version refresh job");
         scheduler::spawn_job(cron_schedule, "refresh compiler versions", move || {
             let fetcher = fetcher.clone();
```

Comment:
can we create new trace here? so we could search for scheduled things later in jaeger

Reply:
(not the span, but a trace)